.github/workflows/stale.yml

```
@@ -6,14 +6,17 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/stale@v6
+    - uses: actions/stale@v8
       with:
         stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed in one week if no further activity occurs. Thank you for your contributions!'
         stale-issue-label: 'stale'
+        stale-pr-message: 'This PR has been automatically marked as stale because it has not had recent activity. It will be closed in one week if no further activity occurs. Thank you for your contributions!'
+        close-pr-message: 'Automatically close due to lack of recent activity, but feel free to reopen with a rebase.'
```

```suggestion
        close-pr-message: 'This PR has been automatically closed due to lack of recent activity, but feel free to reopen it as long as you merge in the main branch afterwards.'
```

It's probably best to be clearer. Also, `git rebase -m main` can also be called a "rebase merge", and Dan and Max tend to use regular merges to update their PRs. It doesn't matter in the end, since it's all squashed anyway, but "merge" might make more sense to newer Git users than "rebase".